controller/admin_controller.py

```diff
@@ -3,10 +3,6 @@
 ###---- Admin Menu ----
 def view_admin_profile(user: User) -> str:
     """
-    Dennis
+    Print information about the user.
```

Comment:
This only prints info about admin users so may make sense to say 'admin user' instead of only user to avoid ambiguity.

Reply:
It actually print info also for any users, that's why I put it this way.

Reply:
Fair enough!